openstack/nova/templates/etc/_nova_audit_map.yaml.tpl

```diff
@@ -58,16 +58,28 @@ resources:
     migrations_legacy_api:
         api_name: os-migrations
         type_uri: compute/migrations
+    # os-floating-ips is deprecated, but it's still being called so it needs to be mapped.
+    os-floating-ips:
+        type_uri: compute/floating-ips
+        el_type_uri: compute/floating-ip
     os-server-external-events:
         type_uri: compute/servers/external-events
         type_name: events
         custom_id: tag
         custom_attributes:
           server_uuid: compute/server
+    # os-security-group-rules is deprecated, but it's still being called, must be mapped.
+    os-security-group-rules
+        type_uri: compute/security-group-rules
+        el_type_uri: compute/security-group-rule
```

Comment:
If I see it right, setting `el_type_uri` is ignored, because it's always computed from `type_uri` https://github.com/sapcc/openstack-audit-middleware/blob/e1d3491d73bc87a5ee56a6e6600fbbc78e296368/auditmiddleware/_api.py#L229-L240

Reply:
I've removed the el_type_uris as they will only be required if they are different